Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -103,6 +103,11 @@ def get_agents_outputs(agents):
             'CreatedAt': agent.get('createdAt'),
             'SiteName': agent.get('siteName'),
         }
+
+        for c in column_to_display:
+            if c in agent:
+                entry[c] = agent.get(c)
```

Comment:
```suggestion
        for c in set(column_to_display).intersection(agent.keys()):
            entry[c] = agent[c]
```

In this suggestion, the iteration is done over a set as opposed to a loop and is more performative.

Another solution would be to store the `column_to_display` argument as a set instead. I'm okay with the current implementation, so it's up to you whether or not you'd like to use this approach.

Reply:
I will update the function with your suggestion as it is more performative ! I realised that I forgot to declare the arg "columns_to_display" as optional as it is used elsewhere without this arg so I will also fix that.